openc3-cosmos-init/plugins/packages/openc3-cosmos-ace-diff/vite.config.js

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  build: {
+    sourcemap: true,
+    rollupOptions: {
+      input: ['./src/index.js', './src/styles/ace-diff.scss', './src/styles/ace-diff-dark.scss'],
+      output: {
+        format: 'commonjs',
```

Comment:
We don't need commonjs output.   Should this be changed?

Reply:
I can probably re-work it to be ESM, but it was commonjs before, so I was just doing the minimal change to get ace-diff on vite

Reply:
Actually yeah, it looks like it'll probably be happy as ESM, which is what Vite wants anyway, so I can undo the changes to admin's vite config

Reply:
I think umd is probably the correct option for now?   Making this esm I don't think would hurt since this one is being bundled, but for tool-common I don't think we want esm yet.   From what I read commonjs is only for node backends.

Reply:
For linked/bundled deps (which I think is only ace-diff), I'd rather use ESM since it's the default for Vite.

tool-common should be systemjs since that's how we're loading it in the app, and then we can build vue-common and js-common out to multiple formats (including systemjs) for publishing in https://github.com/OpenC3/cosmos/pull/1737